subprojects/docs/src/snippets/testing/test-suite-configure-suite-dependencies/groovy/build.gradle

```diff
@@ -36,7 +36,7 @@ testing {
     }
 }
 
-// Note that this is equivalent to:
+// Note that this is equivalent to using the generated "global" configuration name:
 dependencies {
     testImplementation 'org.assertj:assertj-core:3.21.0'
 }
```

Comment:
Maybe instead of showing both options (which might imply you need to do both if you're not really sure what's going on), we could change the example on line 33 to be like:

```
// Note that this is equivalent to using testImplementation in the top-level dependencies block
implementation 'org.assertj:assertj-core:3.21.0' // <2>
```